### PR TITLE
Remove EXIF data from JPEGs

### LINKED ISSUE #54: Delete EXIF data
The privacy policy says

> All images uploaded to OpenSprites have their metadata deleted immediately

We should actually do that if we're not already (server side would be preferred). [This](https://github.com/hMatoba/piexifjs) is a package that looks like it can do the job without any external dependencies.

### PR DESCRIPTION
**Pull request by Tymewalk**
This will strip the EXIF data from JPEGs, fixing #54.

### CODE REVIEW

**Comment by Tymewalk** [comment]
`file = piexif.remove(file)` does not work.

I'm trying to use `exif = piexif.load(file)`, but it keeps saying it's the wrong type.

**Comment by Tymewalk** [comment]
Found one of the big problems - it turns out `file` is not the actual file at all. This is why it assumes nothing is JPEG.

**Comment by Tymewalk** [comment]
Using `file.buffer` gives me the buffer with the proper data in it, and `file.buffer.toString("base64")` makes it possible to use properly.

**Comment by Tymewalk** [comment]
This now works (but still could be improved).

Google around and find an image with a lot of EXIF data in it. Then upload and re-download. It should have no EXIF data in it at all.

**Comment by Tymewalk** [comment]
Just checked. Both images are exactly the same, except the downloaded one has no EXIF data on it. I even opened both in a text editor, same thing.

@MegaApuTurkUltra, how does it look?

File: server/main.js [hunk 1]
```diff
@@ -511,6 +513,15 @@ app.put('/share', upload.single('file'), async function(req, res) {
       res.status(400).json({success: false, message: "Missing file"})
       return
     }
+
+    if(file.mimetype == "image/jpeg") {
+        // Remove EXIF data
+        console.log("User is uploading a JPEG, removing EXIF data")
+        let imageOld = "data:image/jpeg;base64," + file.buffer.toString("base64")
+        let imageNew = piexif.remove(imageOld).substring(imageOld.indexOf(','))
```
Comment (MegaApuTurkUltra) [inline server/main.js:521]: Are you sure this works? Shouldn't it be `indexOf + 1`?

Comment (Tymewalk) [inline server/main.js:521]: @MegaApuTurkUltra 100% sure. Download my repo on that branch and test yourself with any JPEG image (EXIF or not). Works on Firefox 47 for me.

Comment (MegaApuTurkUltra) [inline server/main.js:521]: Alright, seems legit. Is this ready to merge?

Comment (Tymewalk) [inline server/main.js:521]: @MegaApuTurkUltra Ready to merge whenever, unless you'd like that `console.log` removed or changed.

**Pull request merged**